docs/units/standard_units.rst

```diff
@@ -162,7 +162,26 @@ For example::
    Unit(dimensionless with a scale of 1000.0)
    >>> (u.km / u.m).decompose() == u.dimensionless_unscaled
    False
-
+   
+To create a dimensionless quantity with a specific scaling factor, 
+multiply or divide a value by ``Unit()`` with the scaling 
+factor specified, e.g.::
+
+   >>> big_unitless_num = 20000000 # 20 million
+
+   >>> # I will be doing many calculations with big_unitless_num, 
```

Comment:
Generally, I would not put further comments in the example proper; just move it to the main text.